sdcm/cluster.py

```diff
@@ -3490,6 +3487,8 @@ def node_setup(self, node, verbose=False, timeout=3600, wait_db_up=True):  # pyl
             self._scylla_post_install(node, install_scylla)
 
             node.stop_scylla_server(verify_down=False)
+            if not Setup.USE_LEGACY_CLUSTER_INIT:
```

Comment:
Looks like we should clean data unconditionally.

Reply:
done